examples/step-39/step-39.cc

```diff
@@ -194,16 +195,16 @@ namespace Step39
     std::vector<double> boundary_values(fe.n_quadrature_points);
     exact_solution.value_list(fe.get_quadrature_points(), boundary_values);
 
-    const unsigned int deg = fe.get_fe().tensor_degree();
-    const double       penalty =
-      2. * deg * (deg + 1) * dinfo.face->measure() / dinfo.cell->measure();
+    const unsigned int degree = fe.get_fe().tensor_degree();
+    const double penalty = 2. * degree * (degree + 1) * dinfo.face->measure() /
+                           dinfo.cell->measure();
 
     for (unsigned k = 0; k < fe.n_quadrature_points; ++k)
       for (unsigned int i = 0; i < fe.dofs_per_cell; ++i)
         local_vector(i) +=
-          (-fe.shape_value(i, k) * penalty * boundary_values[k] +
-           (fe.normal_vector(k) * fe.shape_grad(i, k)) * boundary_values[k]) *
-          fe.JxW(k);
+          (-fe.shape_value(i, k) * penalty              // -sigma * v_i(x_k)
+           + fe.normal_vector(k) * fe.shape_grad(i, k)) // n * grad v_i(x_k)
```

Comment:
do me a small favor and swap the two factors in `fe.shape_value(i, k) * penalty fe.shape_value(i, k) * penalty` so that they match the order in the comment.

Then in the comments put an opening `(` in front of `-sigma`, a plus in front of the second line, and a closing `)` at the end of the second line. This way comments match the formula.